manager/assets/modext/widgets/resource/modx.panel.resource.js

```diff
@@ -541,7 +541,10 @@ Ext.extend(MODx.panel.Resource,MODx.FormPanel,{
             ,name: 'template'
             ,id: 'modx-resource-template'
             ,anchor: '100%'
-            ,editable: false
+            ,editable: true
+            ,typeAhead: true
```

Comment:
You can continue typing, only if the first matching template is matching from start. 
Add `TemplateTest` to your templates. Then start typing `Templ` and wait for suggestions to pop up, it will preselect `BaseTemplate` and if you'll try to continue typing, you'll get nonsense text.